content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: SelctPanel
+reactId: selectpanel
+figma:
+description: SelectPanel is a semantic dialog that allows for complex selection options within an overlay.
+---
+
+import {Box} from '@primer/react'
+import ComponentLayout from '~/src/layouts/component-layout'
+export default ComponentLayout
+
+## Overview
+
+SelectPanel is used for overlay containers that contain multiple control sets which includes advanced single or multi-selection with a filter. SelectPanel can be anchored to a trigger, or centered within a view.
+
+Commonly, SelectPanel is used for metadata application to an Issue or Pull Request, like assignee and label selections, or the branch selector on a repository’s code tab.
+
+## Anatomy
+
+<img
+  width="960"
+  alt="SelectPanel for assigning labels to an issue. 'Apply labels to this issue' heading, close button in upper right hand corner. Search input with a magnifer icon as a visible label, listbox of label options and the first three are selected with a visual checkbox. The footer contains a button to edit labels, and cancel and apply buttons. Purple labels are overlayed onto SelectPanel: Overlay (dialog) pointing to the container, Search input pointing to the input, role='listbox' bracket of the listbox content, Optional action control to the button in the lower left corner, Explicit confirm / submission buttons bracket for the apply and cancel buttons in the lower right corner."
+  src="https://user-images.githubusercontent.com/40274682/228329313-7c61fff5-0f2f-4170-b319-723e8a2a3a7d.png"
+/>
+
+## Options
+
+<img
+  width="960"
+  alt="First SelectPanel is a branch switcher control that includes tabs, a search input, a listbox with single-select items, and a footer with an All branches link, and cancel and apply buttons. Second SelectPanel is titled 'Show fields, with a list of about 10 multi-select items and apply and cancel buttons in the footer."
+  src="https://user-images.githubusercontent.com/40274682/228330031-030f5dba-19c7-41e1-b4f5-73190775f777.png"
+/>
+<Caption>
+  SelectPanel can have as much as the branch selector component, or as little as just a multi-selection list. If the
+  contents request user input beyond menu selection, it’s likely you need a SelectPanel.
+</Caption>
+
+### Additional control slot
+
+<img
+  width="960"
+  alt="Three cropped SelectPanel footers each displaying a different option of a control in the bottom right left corner: Link, button, or checkbox input."
+  src="https://user-images.githubusercontent.com/40274682/228338168-a9424773-531e-41e3-8d95-261eb971c340.png"
+/>
+<Caption>
+  An optional control slot is available in the SelectPanel’s bottom left corner. Avoid use if not absolutely necessary
+  in order to avoid component and decision overwhelm.
+</Caption>
```

Comment:
```suggestion
<Caption>
  An optional control slot is available in the Select panel’s bottom left corner. Avoid use if not absolutely necessary
  in order to avoid component and decision overwhelm.
</Caption>
```